cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1236,33 +1224,26 @@ function Verify-WorkerServices {
   Log_Todo "run more verification commands."
 }
 
+# Downloads crictl.exe and installs it in $env:NODE_DIR.
 function DownloadAndInstall-Crictl {
-  $CRICTL_VERSION = "v1.17.0"
-  $CRICTL_SHA256 = "781fd3bd15146a924c6fc2428b11d8a0f20fa04a0c8e00a9a5808f2cc37e0569"
-
-  # Assume that presence of crictl.exe indicates that the crictl binaries
-  # were already previously downloaded to this node.
   if (-not (ShouldWrite-File ${env:NODE_DIR}\crictl.exe)) {
     return
   }
-  $tmp_dir = 'C:\crictl_tmp'
-  New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
-
   $url = ('https://storage.googleapis.com/kubernetes-release/crictl/' +
       'crictl-' + $CRICTL_VERSION + '-windows-amd64.exe')
   MustDownload-File `
       -URLs $url `
-      -OutFile $tmp_dir\crictl.exe `
+      -OutFile ${env:NODE_DIR}\crictl.exe `
       -Hash $CRICTL_SHA256 `
       -Algorithm SHA256
+}
 
-  Push-Location $tmp_dir
-  Move-Item -Force crictl.exe ${env:NODE_DIR}\
+# Sets crictl configuration values.
+function Configure-Crictl {
   if (${env:CONTAINER_RUNTIME_ENDPOINT}) {
-    crictl.exe config runtime-endpoint ${env:CONTAINER_RUNTIME_ENDPOINT}
+    & ${env:NODE_DIR}\crictl.exe config runtime-endpoint `
```

Comment:
If ${env:NODE_DIR} path has spaces, should we quote the cmd path? eg. `& "${env:NODE_DIR}\crictl.exe"` https://ss64.com/ps/call.html

Reply:
Good point, done.